core/src/scala/com/mygdx/game/screen/TiledScreen.scala

```diff
@@ -0,0 +1,51 @@
+package com.mygdx.game.screen;
+
+import com.badlogic.gdx.maps.tiled._;
+import com.badlogic.gdx.{ApplicationAdapter, Gdx, Input, InputProcessor, ScreenAdapter};
+import com.badlogic.gdx.graphics.{GL20, OrthographicCamera, Texture};
+import com.badlogic.gdx.graphics.g2d._;
+import com.badlogic.gdx.maps.tiled.renderers.OrthogonalTiledMapRenderer;
+
+import com.mygdx.game.GameApplication
+
+class TiledScreen(game: GameApplication) extends ScreenAdapter{
+
+    var map : TiledMap = _;
+	var renderer : OrthogonalTiledMapRenderer = _;
+
+    val camera = new OrthographicCamera;
+	val font = new BitmapFont;
+	val batch = new SpriteBatch;
```

Comment:
So I have no idea why, but I think the best practice here is to use the already created `SpriteBatch` in `GameApplication`

```scala
val batch = game.batch
// or
game.batch.begin
font.draw(game.batch) // etc..
```